app/generate/route.ts

```diff
@@ -49,26 +60,41 @@ const handleRequest = wrapTraced(async function handleRequest(url: string) {
 
 const getCommits = wrapTraced(async function getCommits(
   owner: string,
-  repo: string,
-) {
-  // Fetch the latest release from the GitHub API
-  const releaseResponse = await octokit.rest.repos.getLatestRelease({
-    owner,
-    repo,
-  });
-  const release = releaseResponse.data;
-  const since = release.published_at;
-
-  // Then fetch the corresponding commits
-  const commitResponse: CommitsResponse = await octokit.rest.repos.listCommits({
-    owner,
-    repo,
-    since: since ?? undefined,
-    per_page: 50,
-  });
+  repo: string
+): Promise<Response | { commits: CommitsResponse['data']; since: string | null }> {
+  try {
+    // Fetch the latest release from the GitHub API
+    const releaseResponse = await octokit.rest.repos.getLatestRelease({
```

Comment:
theoretically we could just return the last 10-20 commits in this case instead of throwing an error, so they still something.

Reply:
done